### DISCUSSION TITLE
Verbose tooltip

### DISCUSSION THREAD

**Discussion by GodBleak** [saadeghi/daisyui#485] [Ideas / feature requests]
I'm currently looking for something like we have here on GitHub, when hovering over a discussion title, you get a tooltip(if it's still called that at that point) that has a lot more detail than one could reasonably have with the current tooltip. Would it be feasible to include such a component into Daisy?

**Comment by saadeghi** [comment]
I think "Dropdown on hover" is what you want:  
https://daisyui.com/components/dropdown/#dropdown-on-hover

**Reply by GodBleak** [reply]
Yup, I think that's a suitable alternative, saves me from having to learn popper, thanks! 

Would be nice if there was an option for a pointer to point to the element it corresponds to, and auto-positioning as I have to remember to position them so they don't get clipped(that's something the tooltip suffers from too), but I totally get(and kind of expect) if those would be out-of-scope.

**Reply by saadeghi** [reply]
Unfortunately auto positioning is not possible with CSS. It needs some JS to check if content is at the corner or not.
